xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd

```diff
@@ -593,7 +593,7 @@ Rail transport, Roads and Road transport
 							<xsd:group ref="PassengerCapacityGroup"/>
 						</xsd:sequence>
 					</xsd:sequence>
-					<xsd:attribute name="id" type="PassengerCapacityIdType" use="optional">
+					<xsd:attribute name="id" type="PassengerCapacityIdType">
```

Comment:
Nope. The default for use is optional.

```
        <xs:attribute name="use" default="optional" use="optional">
```

 Hence you have to make it...
```suggestion
					<xsd:attribute name="id" type="PassengerCapacityIdType" use="required">
```
Apply after you read it ;-)

Reply:
yes sir! won't happen again, sir ;-)